support-frontend/assets/pages/supporter-plus-landing/supporterPlusRouter.tsx

```diff
@@ -41,10 +41,12 @@ const {
 } = store.getState();
 
 const isInTwoStepTest =
-	abParticipations.twoStepCheckout &&
-	abParticipations.twoStepCheckout !== 'control';
-const showCheckoutTopUpAmounts =
-	abParticipations.twoStepCheckout === 'variant_b';
+	abParticipations.twoStepCheckoutWithNudgeBelow &&
+	abParticipations.twoStepCheckoutWithNudgeBelow !== 'control';
+
+console.log(abParticipations);
```

Comment:
@iainjchambers-guardian, I missed this in the review but there's a stray console log here